CMakeLists.txt

```diff
@@ -50,6 +50,10 @@ list(APPEND GAZEBO_SUPPORTED_MODELS "iCubGazeboV2_5")
 list(APPEND GAZEBO_SUPPORTED_MODELS "iCubGazeboV2_5_plus")
 list(APPEND GAZEBO_SUPPORTED_MODELS "iCubGazeboV3")
 
```

Comment:
@ale-git can you provide a short comment explaining what the variable `CONFIGURED_MODELS` contains? From the CMake code it is not obvious that for the models contained in `CONFIGURED_MODELS` the `model.config` for use in Gazebo will be generated, but without generating the helper models `_fixed` and `_feet_fixed`. For people that modify this is in the future, it may be useful to have at least a short comment describing what the `CONFIGURED_MODELS` variable is, or to use a more descriptive name.

Reply:
Hi @traversaro I've added the comment

Reply:
Thanks!